policyDefinitions/SQL/ensure-server-parameter-'audit_log_enabled'-is-on-for-mysql-db-server/azurepolicy.json

```diff
@@ -0,0 +1,110 @@
+{
+  "name": "4dc90661-5d91-41f1-be00-d243f6f3fe9c",
+  "type": "Microsoft.Authorization/policyDefinitions",
+  "properties": {
+    "displayName": "Deploy - Ensure server parameter 'audit_log_enabled' is set to 'ON' for MySQL Database Server",
```

Comment:
Please change the display name as it's confusing that it's says "Deploy...." but allows other effects to.
e.g. Ensure audit log is enabled for MySQL.